src/components/nimbleTab/NimbleTab.tsx

```diff
@@ -19,6 +19,7 @@ interface NimbleTabProps {
   color?: string;
   activeColor?: string;
   tabs: TabData[];
+  activeTabValue: number
```

Comment:
please add this as an optional prop. I guess, if not all the projects need to update after release this